queries.sql

```diff
@@ -72,3 +72,34 @@ SELECT species,
     WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-01-01' 
     GROUP BY species;
 
+/* Multiple tables queries */
+SELECT name AS animal_names, full_name AS owner_name FROM animals 
+JOIN owners ON animals.owner_id = owners.id 
+WHERE owners.full_name = 'Melody Pond';
+
+SELECT animals.name AS animal_names, species.name AS animal_species FROM animals 
+JOIN species ON  animals.species_id = species.id 
+WHERE species.id = 1;
+
+SELECT full_name, name AS animal_names
+FROM animals 
+RIGHT JOIN owners ON animals.owner_id = owners.id;
+
+SELECT species.name, count(*) 
+FROM species 
+JOIN animals ON species.id = animals.species_id 
+GROUP BY species.name;
+
+SELECT name 
+FROM animals 
+JOIN owners ON animals.owner_id = owners.id 
+WHERE owners.full_name = 'Jennifer Orwell' AND animals.name LIKE '%mon';
```

Comment:
- [OPTIONAL]  List all Digimon owned by Jennifer Orwell. It will be better if your can change your syntac since `name = 'Digimon';` and no `%mon` This should be like this 👇🏻 

```sql
   SELECT animals.* FROM animals
   LEFT JOIN owners
   ON animals.owner_id = owners.id
   LEFT JOIN species
   ON animals.species_id = species.id
   WHERE owners.full_name = 'Jennifer Orwell'
   AND species.name = 'Digimon';
```

Reply:
I have also gone ahead to implement the optional changes you pointed out here. Those lines of code look better than they were before, all thanks to you. 💯